<launch>
  <include file="$(find dogsim)/launch/base.launch" >
    <arg name="insitu" value="true"/>
  </include>

  <node pkg="dogsim" type="dog_visualizer" name="dog_visualizer">
    <remap from="dog_position_in" to="/dog_position_detector/dog_position"/>
  </node>
  
  <!-- TODO: Implement 
  <node pkg="rostopic" type="rostopic" name="move_right_arm_away" args="pub r_arm_controller/joint_trajectory_action/goal pr2_controllers_msgs/JointTrajectoryActionGoal '{goal:{trajectory:{joint_names: [l_shoulder_pan_joint, l_shoulder_lift_joint, l_upper_arm_roll_joint, l_elbow_flex_joint, l_forearm_roll_joint, l_wrist_flex_joint, l_wrist_roll_joint], points: [{positions: [1.5, 2.0, 0.0, -2.55, 0.0, -0.1, 0.0], velocities: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], accelerations: [] }]}}}' -1" />
  -->
</launch>
